### ISSUE TITLE
Cannot tab into search results

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#371]
Cannot tab into search results and so cannot to read by screen reader and so cannot be selected 

![screen shot 2016-07-06 at 14 17 27](https://cloud.githubusercontent.com/assets/13200338/16619317/88575cae-4385-11e6-8137-ea4c823165be.png)

**Comment by VitoAlbano** [comment]
depends on : #671

**Comment by hashraf88** [comment]
Unable to test due to #934
